common/unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -153,7 +153,7 @@ public void writeTo(ByteBuffer buffer) {
    *
    * Unlike getBytes this will not create a copy the array if this is a slice.
    */
-  public @Nonnull ByteBuffer getByteBuffer() {
+  @Nonnull public ByteBuffer getByteBuffer() {
```

Comment:
This seems actually a correct order but there is a bug in checkstyle. Please see `https://github.com/checkstyle/checkstyle/issues/903`. I am willing to disable `ModifierOrder` for this method if anyone thinks so.